zebra/zebra_rib.c

```diff
@@ -1599,7 +1599,8 @@ static void zebra_rib_fixup_system(struct route_node *rn)
 }
 
 /* Route comparison logic, with various special cases. */
-static bool rib_compare_routes(const struct route_entry *re1,
+static bool rib_compare_routes(const struct route_node *rn,
```

Comment:
don't see "rn" used?

Reply:
yeah that is left over from some debugging I added.

Reply:
fixed